src/chains/models.py

```diff
@@ -36,6 +36,11 @@ class Chain(models.Model):
     )
     gas_price_oracle_url = models.URLField(blank=True, null=True)
     gas_price_oracle_parameter = models.CharField(blank=True, null=True, max_length=255)
+    gas_price_oracle_gwei_factor = models.PositiveBigIntegerField(
```

Comment:
What happens if one place returns the `gasPrice` in a unit bigger than `gwei`? Shouldn't multiplier be a `float`?

Reply:
Yes good point 👍